scripts/set_config_dword.cmd

```diff
@@ -0,0 +1,11 @@
+@setlocal enabledelayedexpansion
+@echo off
+
+rem  Set a config value (that's a DWORD) in the registry.
+rem  Usage: set_config_dword.cmd <option_name> <new_value>
+
+set "VALUE_NAME=%~1"
+set "VALUE_DATA=%~2"
+
+reg add "HKCU\SOFTWARE\codersnotes.com\Very Sleepy" /v "!VALUE_NAME!" /t REG_DWORD /d !VALUE_DATA! /f
```

Comment:
Why not make the type a parameter instead of having two batch files?

Reply:
I think I did it for readability and to hide the fact that the data was in the registry. The calling scripts care about "config data" not registry stuff.
I can change it if you'd prefer.